src/backend/shared/styles/globals.css

```
@@ -153,6 +153,35 @@
   }
 }
 
+.sidebar-scroll {
+  overflow-y: overlay;
```

Stylelint (17.6.0)

[error] Expected "overlay" to be "auto" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)